README.md

```diff
@@ -27,11 +27,7 @@ After you're finished please remove all the comments and instructions!
 
 <div align="center">
   <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
-
-  <h3><b>Microverse README Template</b></h3>
-
+ 
```

Comment:
- [ ] **Optional:** This is my personal advice, you are not required to do it! However, I noticed that sections such as `Contributing`, `Acknowledgement`, `FAQ` in your Readme are without professional content. I suggest that you take a couple of minutes of your free time to customize them as they do not change much from one project to the other. I hope this is useful. 😃